### ISSUE TITLE
[Task] Files we couldn't receive - Add "showing x - x of x results" span and manage focus

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#121811]
### Feature

document-status

### Team

bmt-team-2 (Bee’s Knees)

### Issue Type

Task

### Summary

Focus needs to be managed when using the pagination on the Files we couldn't receive page

### Description / Context

 When clicking on the pagination results, focus stays on the pagination, and there’s no indication to an assistive technology user that the page has updated the results.

Fix: we should have a line of text right above the cards that says something like “Showing 10-19 of 35 results” and after clicking the pagination buttons focus should move to this line. Match the behavior you see on `track-claims/your-claims`

Note: I don't think that I'm asking too much, but in the case that this is technically hard to implement, I’d suggest moving focus back to the Files not received heading level 2, but it’s not ideal. ALSO: if you do a smooth scroll for the focus, be sure to also include a `prefers-reduced-motion` version that does not scroll, and just "jumps." See: https://github.com/department-of-veterans-affairs/va.gov-team/issues/121816

### Acceptance Criteria / Definition of Done

- [ ] New line telling the veteran which results are showing is added
- [ ] After clicking pagination buttons focus is moved to this results line

### Estimate

3 – Medium sized, a couple of days

### Priority

Highest

**Comment by kirkacrawford-VA** [comment]
@kirkacrawford-VA Connect parent.
